TypeCobol/Compiler/Directives/TypeCobolOptions.cs

```diff
@@ -34,7 +37,18 @@ public bool UseEuroInformationLegacyReplacingSyntax
         }
 
 #if EUROINFO_RULES
+        /// <summary>
+        /// The Instance of the Cpy Copy names Map
+        /// </summary>
+        public CopyNameMapFile CpyCopyNamesMap { get; set; }
```

Comment:
In my opinion should be a private field, and we would have a public `CpyCopyNamesMapFilePath` string property. Load would be performed in the setter of this `CpyCopyNamesMapFilePath` property.

A `Debug.Assert` in `HasCpyCopy(string name)` method would check that either:
- the path is null and then the file is null
- OR the path is not null and then the file is not null too

Reply:
done in e4234a0

Reply:
In the end I kept the property as originally intended, sorry for the misleading remark.